rasa/nlu/model.py

```diff
@@ -447,7 +448,7 @@ def parse(
         data = self.default_output_attributes()
         data[TEXT] = text
 
-        message = Message(data=data, time=time)
+        message = Message(data=data, time=time, time_zone=metadata.get("time_zone"))
```

Comment:
sorry, I don't understand what do you mean

Reply:
@wochinge And then you mean we change the line `self.data.update(**kwargs)` to `self.data.update(metadata)` in `Message` class constuctor?

Reply:
e.g. `intent_metadata` is stored [here](https://github.com/RasaHQ/rasa/blob/80a58261dc033fe1c0a14268d05f216c73e1222d/rasa/shared/nlu/training_data/message.py#L165): `data[METADATA] = {METADATA_INTENT: intent_metadata}`. I'd store the timezone also in `data[METADATA]`, e.g. `data.setdefault(METADATA, {})[ENTITY_METADATA] = example_metadata`

Reply:
Sorry, I didn't completely understand how that would help since we aren't using the class method `build()` but creating a new Message object through it's constructor [here](https://github.com/RasaHQ/rasa/blob/80a58261dc033fe1c0a14268d05f216c73e1222d/rasa/nlu/model.py#L451).

Reply:
I think the `timezone` shouldn't be a top level entry in the `data` dictionary of the `Message` object. In my opinion it should be stored below the `METADATA` key as the other message metadata is as well.